redfish-exporter/docs/metrics.md

```diff
@@ -0,0 +1,59 @@
+# Metrics Documentation
+
+The AMD Redfish Exporter exposes the following Prometheus-compatible metrics:
+
+## Event Metrics
+
+### `RedFishEvents_received`
+
+- **Type**: Counter
+- **Description**: Total number of Redfish events received
+- **Labels**:
+  - `SourceIP`: IP address of the Redfish server that sent the event
+  - `Severity`: Severity of the Redfish event
+
+### `RedFishEvents_processing_time`
+
+- **Type**: Gauge
+- **Description**: Time taken to process Redfish events in seconds
+- **Labels**:
+  - `SourceIP`: IP address of the Redfish server that sent the event
+  - `Severity`: Severity of the Redfish event
+
+### `SlurmAPI_success`
+
+- **Type**: Gauge
+- **Description**: Total number of Slurm API calls that succeeded
+- **Labels**:
+  - `SourceIP`: IP address of the SLURM node
```

Comment:
This is the IP address of redfish server.

Reply:
Gotcha. Fixed!